content/projects/gsoc/2025/project-ideas/improving-tekton-client-plugin.adoc

```diff
@@ -0,0 +1,58 @@
+---
+layout: gsocprojectidea
+title: "Improving Tekton Client Plugin for Jenkins"
+goal: "Enhance the Jenkins Tekton Client Plugin to improve cloud-native CI/CD interoperability and user experience"
+category: Plugins
+year: 2025
+status: draft
+sig: cloud-native
+skills:
+- Java
+- Kubernetes
+- Tekton
+- Jenkins
+mentors:
+- "waveywaves"
```

Comment:
```suggestion
- "waveywaves"
- "krisstern"
```

Maybe you can add me as a co-mentor? I would love to assist and help out if this project idea gets selected for GSoC 2025.